### ISSUE TITLE
`SCIE_BOOT=update scie-pants` & `PANTS_BOOTSTRAP_VERSION=report scie-pants` are clunky.

### ISSUE THREAD

**Issue by jsirois** [pantsbuild/scie-pants#32]
The former leaks an implementation detail. A Pants user should not need to be confronted with what `scie` means. The latter is just plain clunky.

Ideally Pants would support a `bootstrap` pseudo-goal since the namespace has already been taken by `PANTS_BOOTSTRAP_TOOLS` in the existing `./pants` script. That would allow for `scie-pants bootstrap update` and `scie-pants bootstrap version` respectively (or `scie-pants bootstrap --update` and `scie-pants bootstrap --version` to align slightly better with how Pants goals work).

**Comment by jsirois** [comment]
N.B. It seems the scie-pants binary name has converged on `pants` quickly; so modify the above as such; I.E.: `pants bootstrap update` or `pants bootstrap --update`.

**Comment by tdyas** [comment]
Prior art "self" in Rustup: `rustup self update`

**Comment by jsirois** [comment]
Yeah. It's unfortunate we already had the (probably inadvertent) land grab on `bootstrap` via the `PANTS_BOOTSTRAP_TOOLS` introduction. I guess we can just grab more land though and deprecate the bootstrap stuff or leave the world split.

**Comment by kaos** [comment]
Not being well versed in rust, `self` isn't any more obvious than `bootstrap` to me. What I mean is, I think we could go with what feels most natural for Pants regardless of the choice rust has made for them. (i.e. I'm fine with either just that the referenced prior art may not span far outside the rust community.. or maybe it does but I'm unaware)
